prime-angular-frontend/package.json

```diff
@@ -19,68 +19,67 @@
   },
   "private": true,
   "dependencies": {
-    "@angular/animations": "^9.1.0",
-    "@angular/cdk": "^9.2.0",
-    "@angular/common": "^9.1.0",
-    "@angular/compiler": "^9.1.0",
-    "@angular/core": "^9.1.0",
-    "@angular/forms": "^9.1.0",
-    "@angular/material": "^9.2.0",
-    "@angular/material-moment-adapter": "^9.2.0",
-    "@angular/platform-browser": "^9.1.0",
-    "@angular/platform-browser-dynamic": "^9.1.0",
-    "@angular/router": "^9.1.0",
-    "@auth0/angular-jwt": "^3.0.1",
-    "@tinymce/tinymce-angular": "^3.3.1",
-    "@types/smoothscroll-polyfill": "^0.3.1",
-    "@types/tus-js-client": "^1.8.0",
-    "bootstrap": "^4.4.1",
+    "@angular/animations": "^10.2.3",
+    "@angular/cdk": "^10.2.7",
+    "@angular/common": "^10.2.3",
+    "@angular/compiler": "^10.2.3",
+    "@angular/core": "^10.2.3",
+    "@angular/forms": "^10.2.3",
+    "@angular/material": "^10.2.7",
+    "@angular/material-moment-adapter": "^10.2.7",
+    "@angular/platform-browser": "^10.2.3",
+    "@angular/platform-browser-dynamic": "^10.2.3",
+    "@angular/router": "^10.2.3",
+    "@auth0/angular-jwt": "^5.0.2",
+    "@tinymce/tinymce-angular": "^4.1.0",
+    "bootstrap": "^4.5.3",
     "classlist.js": "^1.1.20150312",
-    "core-js": "^3.6.4",
     "filepond": "^4.22.1",
     "filepond-plugin-file-validate-size": "^2.2.1",
     "filepond-plugin-file-validate-type": "^1.2.5",
-    "keycloak-angular": "^7.0.1",
-    "keycloak-js": "^6.0.1",
-    "moment": "^2.24.0",
-    "ng-busy": "^9.0.0",
-    "ngx-clipboard": "^12.2.1",
+    "keycloak-angular": "^8.0.1",
+    "keycloak-js": "^10.0.2",
```

Comment:
@james-hollinger doesn't seem to affect the application bumping the version, but want to draw your eye to the update since if this isn't possible then Keycloak is the only thing holding back upgrading the front-end framework and dependencies.